enable-travis-under-nodejs.md

```diff
@@ -0,0 +1,81 @@
+# How to enable Travis in a repository under the Node.js GitHub organization
+
+Since we don't allow third-party access to the Node.js GitHub organization
+except our own bots, when a repository is transferred into the Node.js
+foundation, Travis won't be able to update the build status of pull requests
+automatically. Instead, we need to use our own GitHub bot to pull the status
+from Travis and send the update in GitHub. This is what needs to be done:
+
+### Step 1: Travis setup
+
+Make sure Travis can listen to the push and pull request events of the
+repository. Go to "Settings -> Integrations & services" page of the
+repository. If Travis has been integrated with the repository before,
+then it should be listed under "Services". If not, click on the dropdown
+"Add service", and search for Travis to add it.
+
+The service does not really have to be configured to enable Travis. All of the
+following configurations work:
+
+* User and Token configured as someone who is not "nodejs", and Domain is
+  blank
+* User and Token left blank, Domain is `notify.travis-ci.org`
+* User and Token configured as someone who is not "nodejs", Domain is
+  `notify.travis-ci.org`
+* User, Token, Domain all left blank
```

Comment:
I do not think this step is working. Or at least it is not working if Travis has not been enabled before on the repo (it's a freshly created repo).
We probably need a guide to enable travis temporarily, flip a switch, then disable.